features/lorawan/mbed_lib.json

```diff
@@ -60,6 +60,10 @@
         "lbt-on": {
             "help": "Enables/disables LBT. NOTE: [This feature is not yet integrated].",
             "value": false
+        },
+        "automatic-uplink-message": {
+            "help": "In case of pending bit, class c confirmed message or sticky MAC command, stack will automatically send empty uplink message",
```

Comment:
Reading the code I can see that even in Class A an automatic uplink can be generated. The help field should therefore be changed.